src/Accelerators/NNPA/Dialect/ZHigh/ZHighOps.td

```diff
@@ -413,20 +414,22 @@ def ZHighLSTMOp:ZHigh_Op<"LSTM", [NoSideEffect,
                        AnyTypeOf<[ZTensor_FICO, NoneType]>:$hidden_bias,
                        SI64Attr:$hidden_size,
                        DefaultValuedStrAttr<StrAttr, "forward">:$direction,
-                       DefaultValuedAttr<SI64Attr, "-1">:$return_all_steps);
+                       DefaultValuedAttr<SI64Attr, "-1">:$return_all_steps,
+                       DefaultValuedStrAttr<StrAttr, "none">:$prev_layer);
```

Comment:
It seems to me it is too early to introduce `prev_layer` at this high level, and it is early to decide the value of `prev_layer`. Operations sandwiched between two LSTMs may be eliminated by optimizations, and the two LSTMs which is not consecutive will become consecutive then.

Reply:
I am deleting prev_layer in ZHigh dialect definition, and implementing a new pass setting the prev_layer argument in zlow.LSTM and zlow.GRU ops.